ui/settingsdlg.ui

```diff
@@ -850,6 +843,29 @@ The qube must be running to disable seamless mode. This setting is not persisten
                 </property>
                </widget>
               </item>
+              <item row="6" column="0">
+               <widget class="QLabel" name="preload_dispvm_label">
+                <property name="text">
+                 <string>Preload disposables:</string>
```

Comment:
I think this needs an explaining tooltip that clarifies what this setting does, exactly.